docs/application/distribute/how-to-distribute/sales.md

```diff
@@ -1,38 +1,40 @@
+  5. In countries where the Tier-method of Phone Bills is supported, when an auto-converted price is less than the standard price of USD 5, the price charged will be the closest Tier price, based on the auto-converted figure (+/- 5% of the auto-converted amount is allowed).
+  6. When you enter a price for applications, an open-lock icon will be shown. When you click the lock-shaped icon to change it to a locked icon, the set price cannot be changed. Also, you cannot change the price for the country when you click the [Auto convert local currency] button. If you need to change the price, simply click on the lock-shaped icon.
+  
+     [![img](media/sales2-330x99.png)](media/sales2.png)
 
 
 ## Availability Date
 
-- Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
 ![img](media/distribution_sales02.jpg)
 
+Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
+
 
 ## Category
 
-- Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
 [![img](media/sales4-630x90.png)](media/sales4.png)
 
+Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
+
 
 ## Age Restriction
 
-- In Age Restriction, select All, 12+, 16+ or 18+.
-- When you select Game as the Category and Korea Republic of or Brazil as the Country for Sales, rating information must be entered. A certificate of age restriction must be issued by an external certification authority. Age restrictions will be approved when the appropriate document is provided. You must therefore upload the appropriate document on age restriction issued from the relevant country. When you upload a GRB certificate for Korea, or MJ/DEJUS certificate for Brazil, select the age restriction that matches the document. If there is no certificate or a certificate cannot be issued, deselect the country from the list of countries where the application will be sold.
- 
-   [![img](media/sales5-730x137.png)](media/sales5.png)
- 
- 
\ No newline at end of file
+![img](media/sales5-730x137.png)](media/sales5.png)
```

Comment:
This image tag is strange.

Reply:
Same as above.

Reply:
The image is shown in this way on github.

![image](https://user-images.githubusercontent.com/9958374/37325722-68691cb6-26d2-11e8-8197-62fd9b878636.png)

Reply:
The left [ was missed. fixed.